sunkit_image/radial.py

```diff
@@ -577,3 +583,116 @@ def fnrgf(
         data[annulus] = ratio_mix[0] * smap.data[annulus] + ratio_mix[1] * data[annulus]
 
     return sunpy.map.Map(data, smap.meta)
+
+
+@u.quantity_input(application_radius=u.R_sun, vignette=u.R_sun)
+def rhef(
+    smap,
+    radial_bin_edges=None,
+    application_radius=0 * u.R_sun,
+    upsilon=0.35,
+    method="numpy",
+    vignette=None,
+    progress=None,
```

Comment:
I think this should default to True instead of None as you default to that in the code below. Then you can remove that if statement.